frontend/public/index.html

```diff
@@ -10,7 +10,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Streamlet</title>
   </head>
-  <body>
+  <body class="dark-mode">
```

Comment:
The dark-mode class used like the default mode, or probably we should get rid of it?